lua/nvim-tree/git/runner.lua

```diff
@@ -138,8 +78,44 @@ function Runner:_run_git_job(callback)
     self:_log_raw_output(data)
   end
 
-  vim.loop.read_start(stdout, vim.schedule_wrap(manage_stdout))
-  vim.loop.read_start(stderr, vim.schedule_wrap(manage_stderr))
+  local untracked = self.list_untracked and "-u" or nil
+  local ignored = (self.list_untracked and self.list_ignored) and "--ignored=matching" or "--ignored=no"
+  local opts = {
+    args = {
+      "--no-optional-locks",
+      "status",
+      "--porcelain=v1",
+      "-z",
+      ignored,
+      untracked,
+      self.path,
+      timeout = self.timeout,
+    },
+    cwd = self.toplevel,
+  }
+
+  log.line("git", "running job with timeout %dms", self.timeout)
+  log.line("git", "git %s", table.concat(utils.array_remove_nils(opts.args), " "))
```

Comment:
Duplicate log.

Reply:
It reports two separate things: timeout and args. Should I merge both into a single line?

Reply:
@alex-courtis That piece comes from the old code (see above). Remove first log?